crypto/keys/src/ristretto.rs

```diff
@@ -110,7 +110,12 @@ impl RistrettoPrivate {
     pub fn to_bytes(&self) -> [u8; 32] {
         *self.0.as_bytes()
     }
-
+    /// This function should be used instead of the from_bytes() function if the
+    /// bytes are **anything** other than the result of calling to_bytes() on a
+    /// RistrettoPrivate.
+    pub fn unchecked_from_bytes(bytes: [u8; 32]) -> Self {
```

Comment:
```suggestion
    pub fn from_bytes_mod_order(bytes: [u8; 32]) -> Self {
```

I'd suggest to call this `from_bytes_mod_order` the sane as the function we're passing through to, because it's easier to remember.

In rust stdlib and core language, usually `unchecked_foo` means an unsafe version of `foo` that will cause undefined behavior in your code if some precondition is not met. Bur that's not what's going on here 

Also you likely need this to be a specific way of turning bytes into scalar, and if it changed, it would break compatibility for your key derivation.

Separately, it might be better to use one of the bytes from hash functions for your use case?